src/index.ts

```diff
@@ -1,12 +1,13 @@
-import express from 'express'
+require('dotenv').config();
+import express from 'express';
 
-const app = express()
-const port = 3000
+const app = express();
+const port = process.env.PORT || 3000;
```

Comment:
imho, I think `??` is more suitable for this case

Reply:
hm, `??` works with nullish values: `null` and `undefined`, whereas `||` would work with any falsy value, which is more important in our case, e.g. `PORT` might be an empty string or `0`, for such cases `||` would give us default value and `??` won't.